### PR TITLE
Init standard UFM Test Service architecture

### PR DESCRIPTION
**Pull request by Spacesai1or**

### CODE REVIEW

**Comment by Spacesai1or** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #7148** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7148" target="_blank">Graphite</a>  👈
    * **PR #7149** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7149" target="_blank">Graphite</a> 
      * **PR #7150** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7150" target="_blank">Graphite</a> 
        * **PR #7151** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7151" target="_blank">Graphite</a> 
          * **PR #7152** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7152" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7148?utm_source=stack-comment).

File: ufm-test-services/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    depends_on:
+      - pushgateway
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+
+  grafana:
```
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:28]: Ignored, grafana needs write

File: ufm-test-services/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+  pushgateway:
+    image: prom/pushgateway
+    ports:
+      - "9091:9091"
+    restart: unless-stopped
+
+  prometheus:
```
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:13]: Updated

File: ufm-test-services/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+  pushgateway:
```
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:4]: Updated

File: ufm-test-services/docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:28]: Updated

File: ufm-test-services/docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:13]: Updated

File: ufm-test-services/docker-compose.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Spacesai1or) [reply ufm-test-services/docker-compose.yml:4]: Updated

File: ufm-test-services/crontab-example [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Needs to point to docker, otherwise you'll get the error: exec: "docker": executable file not found in $PATH
+PATH=/
+
+# Runs every minute
+* * * * * /usr/local/bin/docker-compose -f /path/to/docker-compose.yml --profile 1minute up -d
+
+# Runs every 5 minutes
+*/5 * * * * /usr/local/bin/docker-compose -f /path/to/docker-compose.yml --profile 5minutes up -d
```
Comment (roninjin10) [inline ufm-test-services/crontab-example:8]:
```suggestion
*/5 * * * * /usr/local/bin/docker-compose -f /path/to/docker-compose.yml --profile 5minutes up -d

```
nit: eol

File: ufm-test-services/datesources.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: 1
+
+datasources:
+  - name: Prometheus
+    type: prometheus
+    access: proxy
+    url: http://prometheus:9090
+    isDefault: true
```
Comment (roninjin10) [inline ufm-test-services/datesources.yml:8]:
```suggestion
    isDefault: true

```
nit: eol

File: ufm-test-services/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    depends_on:
+      - prometheus
+    environment:
+      # TODO - Replace with ENV variables to allow for better security
+      - GF_SECURITY_ADMIN_PASSWORD=adminpassword
```
Comment (roninjin10) [inline ufm-test-services/docker-compose.yml:37]:
```suggestion
      # TODO - Replace with ENV variables to allow for better security
      - GF_SECURITY_ADMIN_PASSWORD=${GF_SECURITY_ADMIN_PASSWORD:-adminpassword}
 ```
Can do this immediately

**Review by roninjin10 — APPROVED** [review]
only nits lgtm

**Pull request merged**